src/web_peer_connection.cpp

```diff
@@ -640,7 +643,7 @@ void web_peer_connection::handle_redirect(int const bytes_left)
 
 		// add_web_seed won't add duplicates. If we have already added an entry
 		// with this URL, we'll get back the existing entry
-		web_seed_t* web = t->add_web_seed(redirect_base, web_seed_entry::url_seed, m_external_auth, m_extra_headers);
+		web_seed_t* web = t->add_web_seed(redirect_base, web_seed_entry::url_seed, m_external_auth, m_extra_headers, true);
```

Comment:
could you please explain why these two web seeds should be ephemeral? and confirm that all it means is that they won't be saved in the resume data. (ideally as a brief comment in the code)